### DISCUSSION TITLE
uBO menu doesn't open properly on Tumbleweed

### DISCUSSION THREAD

**Discussion by t-hahn** [uBlockOrigin/uBlock-issues#1801] [Q&A]
I get the following strange appearance of the uBO menu on Tumbleweed (all window managers I tested):
[ublock-menu.mp4](https://wwwth.mpp.mpg.de/members/hahn/tmp/ublock-menu.mp4)
This is on a fresh copy of Firefox 94.0 straight off the Web, with a clean sheet of profiles (`rm -fr ~/.mozilla`).

By no means do I blame uBO for this behavior, yet I am a bit at loss at how to debug it.
Does this look familiar to anybody?  I'm glad for any pointer.
I've looked into fonts (i.e. maybe the horizontal width collapsed to zero if a font wasn't found).  I've looked at the libs Firefox maps, in particular the differences between our system and a fresh copy of Tumbleweed.  But nothing convincing so far.

Thx 1e3,

Thomas

**Comment by ghost** [comment]
Wayland or EGL related?

Maybe one of these: 

- https://bugzilla.mozilla.org/show_bug.cgi?id=1739647
- https://bugzilla.mozilla.org/show_bug.cgi?id=1700340
- https://bugzilla.mozilla.org/show_bug.cgi?id=1678804
- https://bugzilla.mozilla.org/buglist.cgi?quicksearch=ALL%20wayland&list_id=15894948
